documentation/sections/contributing.md

```diff
@@ -0,0 +1,22 @@
+## Contributing 
+Contributions are welcome, feel free to fork a copy and submit a pull request.
+
+If there is a core part of ECMAScript you think we are missing, head on over to [CoffeeScript issues on GitHub](http://github.com/jashkenas/coffeescript/issues), and dig around in the open and closed issues. 
+
+For more resources on adding to CoffeeScript, please see [the Wiki](https://github.com/jashkenas/coffeescript/wiki/%5BHowto%5D-Hacking-on-the-CoffeeScript-Compiler) or [How The Parser Works](https://github.com/jashkenas/coffeescript/wiki/%5BHowTo%5D-How-parsing-works) if you are interested in learning how the parser works. 
+
+There are several things you can do to increase your odds of having the pull request accepted: 
+
+  * Create tests! Any pull request should probably include basic tests to verify you didn't break anything, or future changes won't break your code.
+  * Follow the CoffeeScript style guide
```

Comment:
I can rip this line out.  I was thinking that we probably should have a style guide in place.